extending-valtimo/plugin/custom-plugin-definition.md

```diff
@@ -571,7 +571,7 @@ translation.
 ### Plugin module
 
 Finally, after implementing the components and specification, a module has to be defined for the plugin.
-This module, together with the specification, is then imported in the app module as shown in [on this page](../../valtimo-implementation/modules/core/plugin.md).
+This module, together with the specification, is then imported in the app module as shown in [on this page](valtimo-documentation/getting-started/modules/core/plugin.md).
```

Comment:
i don't think this link will work as the `valtimo-documentation` part is a reference to the repo itself. To make a path from the root of the documentation i believe you can just do `/getting-started/...`